filters/include/pcl/filters/impl/crop_hull.hpp

```diff
@@ -104,16 +104,16 @@ pcl::CropHull<PointT>::getHullCloudRange ()
     -std::numeric_limits<float>::max (),
     -std::numeric_limits<float>::max ()
   );
-  for (std::size_t index = 0; index < indices_->size (); index++)
+  for (pcl::Vertices const & poly : hull_polygons_)
   {
-    Eigen::Vector3f pt = (*input_)[(*indices_)[index]].getVector3fMap ();
-    for (int i = 0; i < 3; i++)
+    for (std::uint32_t const & idx : poly.vertices)
```

Comment:
Should be `auto` instead of `std::uint32_t`, since `vertices` [is `Indices` now](https://github.com/PointCloudLibrary/pcl/blob/master/common/include/pcl/Vertices.h#L19)

Reply:
fixed